app/forms/apo_form.rb

```diff
@@ -101,22 +101,16 @@ def options_for_use_license_type(current_value)
     end.compact # the options block will produce nils for unused deprecated entries, compact will get rid of them
   end
 
-  def default_object_rights
-    return default_rights if model.nil?
-
-    check_default_object_rights || default_rights
-  end
-
-  def check_default_object_rights
-    default_access = model.administrative&.defaultAccess
-
-    return if default_access.nil?
+  def default_access
+    default_access = model&.administrative&.defaultAccess
+    return default_rights if default_access.nil?
+    return 'cdl-stanford-nd' if default_access.controlledDigitalLending
 
     access = default_access.access
-
     return "loc:#{default_access.readLocation}" if access == 'location-based'
-    return 'cdl-stanford-nd' if default_access.controlledDigitalLending
     return "#{access}-nd" if default_access.download == 'none' && access.in?(%w[stanford world])
+
+    access || default_rights
```

Comment:
This line is what fixed the bug.  Essentially, this method was NOT returning the value unless it was in one of the special cases covered by return statements ... so then the form was using "default_rights" which is set in the constructor to be "world".

Reply:
@ndushay Interesting. So... in the bug case, `access` here would be ... `nil`?